azurerm/config.go

```diff
@@ -160,9 +158,12 @@ type ArmClient struct {
 	// Resources
 	managementLocksClient locks.ManagementLocksClient
 
+	// Search
+	searchServicesClient search.ServicesClient
+
 	// ServiceBus
-	serviceBusNamespacesClient    servicebus.NamespacesClient
 	serviceBusQueuesClient        servicebus.QueuesClient
+	serviceBusNamespacesClient    servicebus.NamespacesClient
```

Comment:
why change the order here?

Reply:
I'd noticed this wasn't sorted by name (I'm trying to keep the sub-groups sorted alphabetically, just to keep it easier to skim, there's no technical reason)